package.json

```diff
@@ -10,19 +10,19 @@
     "compile": "hardhat compile",
     "test": "hardhat test",
     "coverage": "hardhat coverage",
-    "deploy": "hardhat deploy --network rinkeby",
+    "deploy:mainnet": "hardhat deploy --network mainnet",
+    "deploy:rinkeby": "hardhat deploy --network rinkeby",
```

Comment:
theoretically the network could be a parameter: see https://github.com/gnosis/safe-contracts/blob/main/package.json#L23

You then could write `hardhat deploy mainnet` (or any other network configured)